common/src/IO/SystemPaths.cpp

```diff
@@ -41,10 +41,10 @@ namespace TrenchBroom {
             }
             
             Path resourceDirectory() {
-#if defined __APPLE__
-                return IO::Path(wxStandardPaths::Get().GetResourcesDir().ToStdString());
-#else
+#if defined _WIN32
                 return appDirectory() + Path("Resources");
```

Comment:
We should leave the CPack config as it is. According to [wxStandardPaths::GetResourcesDir()](http://docs.wxwidgets.org/trunk/classwx_standard_paths.html#a5514bf6288ee9f5a0acaf065762ad95d), the resources should reside directly inside the application directory anyway.